### DISCUSSION TITLE
how to auto generate code with Brace Brackets

### DISCUSSION THREAD

**Discussion by Strypper** [cake-build/cake#4115] [Q&A]
The result I want
```xml
<Frame Style="{x:StaticResource DocumentContentFrameStyle}">
                <Label Text="{x:StaticResource ControlInfo}" />
</Frame>
```

What I provide to dotnet cake 

```xml
<Frame Style=""{"x:StaticResource DocumentContentFrameStyle"}"">
                <Label Text=""{"x:StaticResource ControlInfo"}"" />
</Frame>
```

What it generate for me 
```xml
<Frame Style="x:StaticResource DocumentContentFrameStyle">
                <Label Text="x:StaticResource ControlInfo" />
</Frame>
```

**Comment by augustoproiete** [comment]
Hey @Strypper, could you provide a reproducible example of the above? Cake is a build orchestrator, so ultimately it calls `dotnet build`, or `msbuild`, or some other tool that then is responsible for any code generation for what it looks like it is WPF code, so we need to understand what is being executed from your Cake script.
